lib/Elasticsearch/Tests/Functional/FunctionalElasticsearchTestCase.php

```diff
@@ -243,18 +250,27 @@ protected function configureConditionGenerator(QueryConditionGenerator $conditio
             ]
         );
 
+        $conditionGenerator->registerField('@customer-regdate', '/customers/customers#reg_date', [
+            '#type' => 'customer',
```

Comment:
Note, we're adding a conditional condition to the order field, it allows us to specify which documents to even consider for sorting (for example, skip the ones which don't have the field we're ordering by).